makefile

```diff
@@ -1,46 +1,57 @@
-.DEFAULT_GOAL := all
-.PHONY: all release clean
-TODAY ?=$(shell date +%Y-%m-%d)
-VERSION=v$(TODAY)
-SOURCE_URL=https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
-
-
-# MAIN COMMANDS / GOALS ------------------------------------------------------------------------------------------------
-all: tmp/output/release/icd11foundation.owl
-
-clean:
-	rm -rf tmp/
-
-tmp/output/release/icd11foundation.owl: tmp/output/intermediate1-unicode-cleaned.owl
-	robot remove --axioms equivalent -i $< -o $@
-
-# Cleans all unicode characters
-tmp/output/intermediate1-unicode-cleaned.owl: tmp/input/source.owl | tmp/output/release/
-	tr -cd '\11\12\15\40-\176' < $< > $@
-
-tmp/output/release/:
-	mkdir -p $@
-
-tmp/input/:
-	mkdir -p $@
-
-tmp/input/source.owl: tmp/input/source.gz
-	gunzip -c $< > $@
-
-tmp/input/source.gz: | tmp/input/
-	wget ${SOURCE_URL} -O $@
-
-# Requires GitHub CLI: https://cli.github.com/
-release: | tmp/output/release/
-	@test $(VERSION)
-	gh release create $(VERSION) --notes "New release." --title "$(VERSION)" tmp/output/release/*
-
-# HELP -----------------------------------------------------------------------------------------------------------------
-help:
-	@echo "-----------------------------------"
-	@echo "	Command reference: GARD OWL Ingest"
-	@echo "-----------------------------------"
-	@echo "all"
-	@echo "Runs ingest and creates all release artefacts.\n"
-	@echo "release"
-	@echo "Uploads release to GitHub.\n"
+.DEFAULT_GOAL := all
+.PHONY: all release clean
+TODAY ?=$(shell date +%Y-%m-%d)
+VERSION=v$(TODAY)
+SOURCE_URL=https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
+
+
+# MAIN COMMANDS / GOALS ------------------------------------------------------------------------------------------------
+all: tmp/output/release/icd11foundation.owl
+
+clean:
+	rm -rf tmp/
+
+tmp/output/release/icd11foundation.owl: tmp/output/intermediate1-unicode-cleaned.owl
+	robot remove --axioms equivalent -i $< -o $@
+
+# Cleans all unicode characters
+tmp/output/intermediate1-unicode-cleaned.owl: tmp/input/source.owl | tmp/output/release/
+	tr -cd '\11\12\15\40-\176' < $< > $@
+
+tmp/output/release/:
+	mkdir -p $@
+
+tmp/input/:
+	mkdir -p $@
+
+tmp/input/source.owl: tmp/input/source.gz
+	gunzip -c $< > $@
+	rm $<
+
+tmp/input/source.gz: | tmp/input/
+	wget ${SOURCE_URL} -O $@
+
+# Requires GitHub CLI: https://cli.github.com/
+release: | tmp/output/release/
+	@test $(VERSION)
+	gh release create $(VERSION) --notes "New release." --title "$(VERSION)" tmp/output/release/*
+
+# Mappings
+tmp/input/mappings.json: | tmp/input/
+	wget -qO tmp/input/en_product1.json.tar.gz https://www.orphadata.com/data/json/en_product1.json.tar.gz
+	tar -xvf tmp/input/en_product1.json.tar.gz -C tmp/input/
+	mv tmp/input/en_product1.json $@
+	rm tmp/input/en_product1.json.tar.gz
+
+tmp/output/release/ordo-icd11.sssom.tsv: tmp/input/mappings.json | tmp/output/release/

```

Comment:
## How to know if the ICD11 reference is to the _foundation_ (not a linearization, like MMS)

Julie states in email "Orphanet mappings":
> ICD-11 is included into ORDO (release every 6 months) but only the MMS code is represented. If you need URI (from the Foundation) and MMS aligned codes, you need to download the alignment file on Orphadata or the nomenclature pack.

If I `grep` against the website download, I see 3715 instances of this note, but I don't know precisely what it's trying to say yet:
> "label": "Index term (ICD-10: Orphanet entity listed in the ICD-10 Index. ICD-11: Orphanet entity listed in the ICD-11 Foundation)"

I've been in discussion with Julie over e-mail, but she iterates that even though I'm seeing this string of text, it is not indicative that the term is a Foundation term. It sounds like I need to get it from the nomenclature pack, but I don't know how to get that yet (I only see one for SNOMED).


### More example code snippets
I did not find any instances of the above string with the word "Foundation" in the downloaded JSON from the alignment page. But I do see them in http://www.orphadata.org/data/ORDO/ordo_orphanet.owl.

<details><summary>Example axioms w/ the word 'Foundation'</summary>
<p>

```owl
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://www.orpha.net/ORDO/Orphanet_100086"/>
        <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasDbXref"/>
        <owl:annotatedTarget>ICD-11:2C13.Y</owl:annotatedTarget>
        <oboInOwl:source xml:lang="en">- NTBT (ORPHAcode is narrower than the targeted code used to represent it).
- Index term (ICD-10: Orphanet entity listed in the ICD-10 Index. ICD-11: Orphanet entity listed in the ICD-11 Foundation).</oboInOwl:source>
    </owl:Axiom>
```

</p>
</details> 

<details><summary>Example axioms w/out the word 'Foundation'</summary>
<p>

```owl
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://www.orpha.net/ORDO/Orphanet_100081"/>
        <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasDbXref"/>
        <owl:annotatedTarget>ICD-10:C20</owl:annotatedTarget>
        <oboInOwl:source xml:lang="en">- NTBT (ORPHAcode is narrower than the targeted code used to represent it).
- Attributed code (ICD-10/ICD-11: The targeted code is assigned by Orphanet).</oboInOwl:source>
    </owl:Axiom>
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://www.orpha.net/ORDO/Orphanet_100081"/>
        <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasDbXref"/>
        <owl:annotatedTarget>ICD-11:2B92.1</owl:annotatedTarget>
        <oboInOwl:source xml:lang="en">- E (Exact mapping: the two concepts are equivalent).
- Specific code (ICD-10/ICD-11: ORPHAcode has its own code in the targeted terminology).</oboInOwl:source>
    </owl:Axiom>
```

</p>
</details>